src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -920,7 +931,11 @@ private void addDelegate(final AuthenticationManager delegate, final String conf
 
         public Authentication authenticate(Authentication authentication) throws AuthenticationException {
             if (delegates.size() == 1) {
-                return updateUserDetails(delegates.get(0).delegate.authenticate(authentication));
+                try {
+                    return updateUserDetails(delegates.get(0).delegate.authenticate(authentication));
+                } catch (AuthenticationServiceException e) {
+                    LOGGER.log(Level.WARNING, "Failed communication with ldap server.", e);
```

Comment:
But you are continuing anyway? That is more than a logging change.

Reply:
I've updated the code to rethrow the exception. (It seems GitHub doesn't recognize the comment as outdated because the change is on the next line)